docs/guides/migration.md

```diff
@@ -0,0 +1,148 @@
+---
+page_title: "Migrating from the B1DDI provider"
+subcategory: "Guides"
+description: |-
+  This guide outlines the migration process from the B1DDI provider to the BloxOne provider.
+---
+
+# Migrating from the B1DDI provider
+
+This guide covers the changes introduced in the BloxOne provider and outlines the steps you may need to take to upgrade your configuration.
+
+The BloxOne provider replaces the [B1DDI provider](https://registry.terraform.io/providers/infobloxopen/b1ddi/latest) and is not backwards compatible. This means you will need to update your configuration to use the new provider.
+
+## Backup
+
+Before making changes to your state, it's a good idea to back up your state file. Any state modification commands made using the CLI will automatically create a backup. 
+If you prefer to manually back up your state file, you can copy your `terraform.tfstate` file to a backup directory.
```

Comment:
Do we have to mention copying to a backup directory? Usually, the versioning would be maintained using a SCM tool like GitHub

Reply:
The backup for terraform state works automatically usually with `.backup` files generated every time a state modification command is run. Usually `.tfstate` and `.tfstate.backup` are not managed in Git, because it contains sensitive files. They are usually saved as a remote state in s3 etc. 

The manually copy part here is not necessary, but since the backup files can have a number of timestamps, it can be useful to just copy the `.tfstate` file and store it with another name or another directory.